components/layout_2020/display_list/stacking_context.rs

```diff
@@ -1366,17 +1372,70 @@ impl BoxFragment {
         Some(display_list.define_clip_chain(*parent_clip_chain_id, [clip_id]))
     }
 
-    fn build_scroll_frame_if_necessary(
+    fn used_overflow(&self) -> (ComputedOverflow, ComputedOverflow) {
+        let overflow = self.style.effective_overflow();
```

Comment:
Maybe `effective_overflow()` should accept a `FragmentFlags`, and handle replaced elements there?

Reply:
Then at least return a `PhysicalVec<ComputedOverflow>` to make it easier to pass around.